dask/base.py

```diff
@@ -150,6 +185,44 @@ def compute(*args, **kwargs):
 
 
 def visualize(*args, **kwargs):
```

Comment:
On Mon, Jun 6, 2016 at 4:37 PM Jim Crist <notifications@github.com> wrote:

> In dask/base.py
> <https://github.com/dask/dask/pull/1234#discussion_r65974073>:
>
> > @@ -150,6 +185,44 @@ def compute(*args, **kwargs):
> >
> >
> >  def visualize(*args, **kwargs):
>
> Variadic positional here feels a bit nicer as an api (no need to build a
> list, and works with single and many arguments trivially), and you can't
> have named keywords with variadic positional (def foo(*args,
> some_name=None): is a syntax error).
>
Ah, yes. Now I've come to take Python 3 for granted... :)

Nice for everyone but first time users, but I don't feel strongly about it.
Just have to read the docs.

>